### ISSUE TITLE
keep icon when changing favourite group

### ISSUE THREAD

**Issue by pebogufi** [osmandapp/OsmAnd#14245]
### Description
currently if i change (move) a favourite from one group to a different, color and icon are changed to that of new group.

### Describe the solution you'd like
Please 
- either keep the old icon
or
-  add the old icon to icon history, so that i can get it from there and can change it back.

**Comment by tradavyvy** [comment]
This 'problem' is linked to the now closed issue #10850 
Part of the suggested solution in this issue was to keep icon, colour and shape of a favourite fixed if it was set manualy, and change it to the defaults of its new group if it had not been set manualy.
Apparently at least part of #10850 has been implemented in the upcomming 4.2 version.
If this part has not, maybe an algoritm like this might work:
-If the favourite being moved or copied has a color/icon/shape in its setting
-Then ignore
-Else use current group setting (but do'nt set this favorites settings)

In other words, a favourite by default has no icon/colour/shape setting, but it will use the current setting of its group. Unless there is a (mannual) setting in the ispecific settings of the favorite itself, then that setting has priority.

(A reset/clear the specific setting of this favorite choice would be a nice option in that case. To make sure this favourite 'follows' its group setting from now on.)

**Comment by scaidermern** [comment]
Why not ask the user what to do in this case? Keeping the appearance or applying the default appereance of the group.
